pkgs/by-name/sb/sby/package.nix

```
@@ -39,24 +39,35 @@ stdenv.mkDerivation rec {
   ];
 
   patchPhase = ''
```

but the better solution is 
```suggestion
  postPatch = ''
```